demo/ssn-input.html

```diff
@@ -27,15 +28,23 @@
         width: auto;
         text-align: center;
       }
+
+      .container {
+        @apply(--layout-horizontal);
+      }
     </style>
 
     <ssn-validator></ssn-validator>
 
-    <input is="iron-input" maxlength="3" bind-value="{{_ssn1}}" size="3" aria-label="First 3 digits of social security number">
-    -
-    <input is="iron-input" maxlength="2" bind-value="{{_ssn2}}" size="2" aria-label="Middle 2 digits of social security number">
-    -
-    <input is="iron-input" maxlength="4" bind-value="{{_ssn3}}" size="4" aria-label="Last 4 digits of social security number">
+    <div class="container">
+
+      <input is="iron-input" maxlength="3" bind-value="{{_ssn1}}" size="3" aria-label="First 3 digits of social security number">
```

Comment:
I think you can just set a style for  `input { width: auto; }` to undo the `100%`, and don't need `iron-flex-layout`